docs/content/guide/unit-testing.ngdoc

```diff
@@ -20,13 +20,13 @@ related pieces such as the DOM elements, or making any XHR calls to fetch the da
 
 While this may seem obvious it can be very difficult to call an individual function on a
 typical project. The reason is that the developers often mix concerns resulting in a
-piece of code which does everything. It makes an XHR request, it sorts the response data and then it
+piece of code which does everything. It makes an XHR request, it sorts the response data, and then it
 manipulates the DOM.
 
-With Angular we try to make it easy for you to do the right thing, and so we
-provide dependency injection for your XHR requests, which can be mocked, and we provide abstractions which
-allow you to test your model without having to resort to manipulating the DOM. The test can then
-assert that the data has been sorted without having to create or look at the state of the DOM or
+With Angular, we try to make it easy for you to do the right thing. For your XHR requests, we
```

Comment:
`XHR requests` is redundant, speaking of tighter english =) But I think it's okay anyways.